nft/nfts.yaml

```diff
@@ -740,27 +739,21 @@ paths:
                         sellerAddress:
                           type: string
                           description: String - The address of the seller in the NFT sale.
-                        buyerIsMaker:
-                          type: boolean
-                          description: Boolean - True if the buyer was the maker (put out bid), false if the buyer was the taker (accepted seller listing) in the order.
-                        price:
-                          type: float
-                          description: Float - The amount paid to the NFT seller in the sale.
-                        priceCurrency:
+                        taker:
                           type: string
-                          description: String - The token used to pay the seller in the sale.
-                        protocolFee:
-                          type: float
-                          description: Float - The amount paid to the marketplace in the sale.
-                        protocolFeeCurrency:
-                          type: string
-                          description: String - The token used to pay the marketplace in the sale.
+                          description: String - Whether the taker in the trade was the buyer or the seller.
+                          enum:
+                            - BUYER
+                            - SELLER
+                        sellerFee:
+                          type: object
+                          description: The amount, token symbol, and decimals of the token of the payment from buyer to seller
```

Comment:
is seller fee the right name here or is sellerPrice better? fee makes me think it's an additional charge on top of the price of the NFT, not the actual total cost of the NFT which is what this seems like it is.

Reply:
ahh i see below there are three components into the total cost - do we also want to add a variable for total cost which is the sum of these three (marketplaceFee + royaltyFee + sellerFee)

or does the royaltyFee come out of the sellerFee?

Reply:
The total price is the sum of these three values, but it's kind of annoying that in Seaport the fees don't have to be the same token. I would like to have something like `totalUSDPrice` in the future, but for now could add a `totalPrice` field that is null if the fee tokens differ.

Reply:
totalUSDPrice makes sense in the future